### ISSUE TITLE
totw bugs GoT and curator

### ISSUE THREAD

**Issue by Splooder24** [project-rubika/issues#503]
**totw some npc's won't die Guardian of tomorrow and also wanders around getting stuck,The Curator aggro when you are close to the room he is in**

**To Reproduce**
4. See error or Notice this behavior

**Expected behavior**

**Screenshots**

**Character info:**
lvl 55 enf name Letssmash
```
Copy/paste the output of /getfull here
```

**Playfield info:**
totw
```
Copy/paste the output of Shift+F9 here
```
----

 
![guard](https://github.com/project-rubika/issues/assets/156912042/ae0eca93-bec6-4299-b381-eb6bbdb8049c)

**Comment by prkdrake** [comment]
Closed as duplicate of #393

**Comment by Splooder24** [comment]
i will be more careful to post bugs cuz it must be exhausting to check all this
